gradle/dependencies.gradle

```diff
@@ -18,3 +18,8 @@ libraries.bytebuddyandroid = "net.bytebuddy:byte-buddy-android:${versions.bytebu
 libraries.objenesis = 'org.objenesis:objenesis:2.6'
 
 libraries.asm = 'org.ow2.asm:asm:6.0'
+
+libraries.kotlin = [:]
```

Comment:
To reduce the duplication, you just do:

```
libraries.kotlin = [
  stdlib: "...",
  ...
]
```

Reply:
Hm, I edited online on GitHub, but it seems to be broken now 😢